packages/core/src/external/commonwell/contribution/shared-document-retrieval.ts

```diff
@@ -0,0 +1,31 @@
+import { parseFilePath } from "../../../domain/filename";
+import { S3Utils } from "../../../external/aws/s3";
+
+export interface DocumentRetrievalParams {
+  fileName: string;
+  s3Utils: S3Utils;
+  bucketName: string;
+}
+
+export async function retrieveDocumentForCommonWell(
```

Comment:
Can we rename this to something that explicitly mentions contribution, shareback, and/or inbound, please? Reading this function name I think we're executing outbound DR on CW.